src/plugins/GUITestBase/src/base_dialogs/GTFileDialog.cpp

```diff
@@ -323,6 +329,19 @@ void GTFileDialogUtils::setViewMode(ViewMode v) {
 }
 #undef GT_METHOD_NAME
 
+#define GT_METHOD_NAME "applyFilter"
+void GTFileDialogUtils::applyFilter() {
+    if (filter.isEmpty()) {
+        return;
+    }
+
+    QComboBox* comboBox = fileDialog->findChild<QComboBox*>(FILE_TYPE_COMBO_BOX);
```

Comment:
Use GTWidget::findComboBox

Reply:
This function doesn't exist, but I added it